src/view/components/interface.ts

```diff
@@ -0,0 +1,117 @@
+import type { BaseItemDto } from '@jellyfin/sdk/lib/generated-client';
+
+export interface QueryI extends ViewSettingsI {
```

Comment:
Can we exclude the `I` prefixes and suffixes? I've never been a fan of that convention in Java either. :sweat_smile:

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/d7e48d30b6480d3af9083b3aa36eebb4331a2d3f